classes/class-uagb-rest-api.php

```diff
@@ -90,24 +90,30 @@ public function content_pre_render( $block ) {
 			if ( empty( $block_css_style ) ) {
 				return $block;
 			}
+		
+			if ( ! empty( $block['attrs'] ) && is_array( $block['attrs'] ) ) {
+				// This line of code creates a new array named $font_family_attrs by searching through the keys of an existing array.
+				$font_family_attrs = preg_grep( '/fontfamily/i', array_keys( $block['attrs'] ) );
+				$link_tag_list     = '';
+
+				if ( empty( $font_family_attrs ) && ! is_array( $font_family_attrs ) ) {
+					return $block;
```

Comment:
Move foreach inside this if check. so that we can create style.
```suggestion
					if( ! empty( $font_family_attrs ) &&  is_array( $font_family_attrs ) )
```